src/rules/export.js

```diff
@@ -1,7 +1,8 @@
 import ExportMap, { recursivePatternCapture } from '../ExportMap';
 import docsUrl from '../docsUrl';
-import includes from 'array-includes';
-import flatMap from 'array.prototype.flatmap';
+
+const includes = Function.bind.bind(Function.prototype.call)(Array.prototype.includes);
```

Comment:
How about extract them into a seperate file for reusing? (Except the `webpack` resolver)

Reply:
Creating a new file might introduce merge conflicts in the future though. Do we really need to do this?

Reply:
> Creating a new file might introduce merge conflicts in the future though

How? A same file name?

Reply:
> > Creating a new file might introduce merge conflicts in the future though
> 
> How? A same file name?

Yeah. I am afraid of that.

Also, `eslint-plugin-import` requires additional compilation before publishing. So less file changes the better.